options/options.go

```diff
@@ -991,6 +1010,11 @@ func findOptionNode[N ast.Node](
 	var bestMatchLen int
 	nodes.Range(func(node N, val ast.ValueNode) bool {
 		srcInfo := srcInfoAccessor(node)
+		if srcInfo == nil {
+			// can happen if we are lenient when interpreting -- this node
+			// could not be interpreted and thus has no source info; skip
+			return true
+		}
```

Comment:
THIS is the actual fix for the panic. In the cases where `srcInfo` was nil here, it would trigger a nil-dereference panic on the next line, to look at the `Path` field. Oops.